### PR TITLE
Add 'users policy'

### PR DESCRIPTION
**Pull request by harshavardhana**

### CODE REVIEW

File: cmd/admin-users-add-policy.go [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+	// Get the alias parameter from cli
+	args := ctx.Args()
+	aliasedURL := args.Get(0)
+
+	// Create a new Minio Admin Client
+	client, err := newAdminClient(aliasedURL)
+	fatalIf(err, "Cannot get a configured admin connection.")
+
+	fatalIf(probe.NewError(client.SetUserPolicy(args.Get(1), args.Get(2))).Trace(args...), "Cannot set user policy for new user")
```
Comment (poornas) [inline cmd/admin-users-add-policy.go]: might be better to reword as ```Cannot set user policy for user``` as the mc admin users add command also allows overwrite

File: cmd/admin-users-add-policy.go [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+import (
+	"github.com/fatih/color"
+	"github.com/minio/cli"
+	"github.com/minio/mc/pkg/console"
+	"github.com/minio/mc/pkg/probe"
+)
+
+var adminUsersPolicyCmd = cli.Command{
+	Name:   "policy",
+	Usage:  "Add policy for user",
```
Comment (kannappanr) [inline cmd/admin-users-add-policy.go]: Can this be `Set policy for user`?

File: cmd/admin-users-add-policy.go (hunk 1 above)
Comment (harshavardhana) [reply cmd/admin-users-add-policy.go]: fixed

File: cmd/admin-users-add-policy.go (hunk 2 above)
Comment (harshavardhana) [reply cmd/admin-users-add-policy.go]: fixed

**Pull request merged**